cmd/serve.go

```diff
@@ -24,6 +25,15 @@ Use the --port and --host flags to change the listening option.
 If you are the developer of a client-side app, you can use --appdir
 to mount a directory as the application with the 'app' slug.
 `,
+	Example: `The most often, this command is used in its simple form:
+
+	$ cozy-stack serve
+
+But if you want to develop two apps in local (to test their interactions for
+example), you can use the --appsdir flag like this:
+
+	$ cozy-stack serve --appsdir appone:/path/to/app_one,apptwo:/path/to/app_two
```

Comment:
@nono Does it still works with another app (like Files) installed directly via a POST on `/apps/files` ?